packages/firestore/src/model/field_value.ts

```diff
@@ -492,6 +541,11 @@ export class GeoPointValue extends FieldValue {
     }
     return this.defaultCompareTo(other);
   }
+
+  byteSize(): number {
+    // GeoPoints are made up of two distinct numbers (latitude + longitude)
+    return 8;
```

Comment:
Should perhaps be 16 then, I think... although JavaScript may use some clever encoding of numbers that allows it to use 4-bytes for small numbers?